### PR TITLE
journal+ journalsheet added

### PR DESCRIPTION
**Pull request by simonriedmann**
Journal html structure added, created journalsheet.html

### CODE REVIEW

File: src/journal.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h1>Journal</h1>
+    </header>
+    <main>
+        <button>Rate today</button>
```
Comment (KarinLiegmann) [inline src/journal.html]: Please add the white star after the button

File: src/journal.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            <p>"Thats life in the city"</p>
+            <h3>Notes:</h3>
+            <p>Si sine causa? quae fuerit causa, mox videro; interea hoc tenebo, si mihi. Et quidem se repellere, idque instituit docere sic omne animal, simul atque.</p>
+
+        </article>
+        <article>
+            <h2>12.07.2020</h2>
+            <h3>Rating:</h3>
+
+            <h3>Comprehension:</h3>
```
Comment (KarinLiegmann) [inline src/journal.html:203]: Please add the beautiful rectangles :)

File: src/journal.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+            <h3>Motto:</h3>
+            <p>"Thats life in the city"</p>
+            <h3>Notes:</h3>
+            <p>Si sine causa? quae fuerit causa, mox videro; interea hoc tenebo, si mihi. Et quidem se repellere, idque instituit docere sic omne animal, simul atque.</p>
+
+        </article>
+        <article>
+            <h2>12.07.2020</h2>
+            <h3>Rating:</h3>
+
```
Comment (KarinLiegmann) [inline src/journal.html:202]: Please add the stars

File: src/journalsheet.html [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                    </g>
+                </g>
+            </svg>
+        </div>
+        <h3>Motto:</h3>
+        <textarea name="motto" id="" cols="30" rows="10"></textarea> 
+        <h3>Notes</h3>
+        <textarea name="notes" id="" cols="30" rows="10"></textarea>
+        <button>Cancel</button>
+        <button>Save</button>
```
Comment (KarinLiegmann) [inline src/journalsheet.html]: Please add icon after text

**Review by KarinLiegmann — CHANGES_REQUESTED** [review]

**Review by GitHogg — APPROVED** [review]
nice

**Review by GitHogg — APPROVED** [review]
also nice

**Pull request merged**